integration_tests/sdk/conftest.py

```diff
@@ -2,34 +2,20 @@
 from typing import List
 
 import pytest
-import utils
 from aqueduct.dag import DAG, Metadata
 
 import aqueduct
 
-# Usage: add a <flag> in FLAGS which will enable `--{flag}` in test cmd options.
-# The option variable can be accessed through utils.flags during tests.
-# One can also mark test to be triggered only when `--{flag}` is turned on through
-# @pytest.mark.<flag_name>
-FLAGS = []
-
 
 def pytest_addoption(parser):
-    for flag in FLAGS:
-        parser.addoption(f"--{flag}", action="store_true", default=False)
+    # We currently only support a single engine backend at a time.
+    parser.addoption(f"--engine", action="store", default=None)
 
 
 API_KEY_ENV_NAME = "API_KEY"
 SERVER_ADDR_ENV_NAME = "SERVER_ADDRESS"
 
 
-@pytest.fixture(autouse=True)
```

Comment:
Why are we getting rid of this fixture?

Reply:
all this fixture did was stick the flags onto the global var in `utils.py`. We don't use this variable anymore so deleting.